<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>620</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Pogoda</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabMapType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_gdal">
     <attribute name="title">
      <string>GDAL</string>
     </attribute>
     <widget class="QPushButton" name="bLayerUp">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>▲</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerDown">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>▼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerNew">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Nowa warstwa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerVisibility">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Widoczność</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerDelete">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń warstwę</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Warstwy</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_osm">
     <attribute name="title">
      <string>OpenStreetMap</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>pobierz mapę</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>lon:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>lat:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerUp_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>▲</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerDown_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>▼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerVisibility_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Widoczność</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLayerDelete_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuń warstwę</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Warstwy</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_MapType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetlana mapa:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNavUp">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▲</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNavDown">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>▼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNavRight">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>►</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNavLeft">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>◄</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_textScale">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScale">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bNavUp_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bNavUp_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupWeather">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Ustawienia pogody</string>
    </property>
    <widget class="QComboBox" name="comboHour">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_hour">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Godzina:</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>216</width>
       <height>155</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tryb zaawansowany</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupAdvanced">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_param">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parametr:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboParam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkFilter">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Filtr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_color">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Kolory:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bColor1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bColor2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupFilter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineHigher">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineLower">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="bShowWeather">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyświetl</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupGraph">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Przebieg wybranego parametru w czasie</string>
    </property>
    <widget class="QLabel" name="label_graph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Okres czasu:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboGraph">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsGraph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>931</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Współrzędne: xxxxxxxxx N, xxxxxxxxxxxx E;   Temperatura: 21oC, Wilgotność powietrza: 40%, Opady: brak, Prędkość wiatru: 25km/h, Kierunek: NE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPogoda">
    <property name="title">
     <string>Opcje</string>
    </property>
    <addaction name="actionUstawienia_programu"/>
    <addaction name="actionWyj_cie"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionO_programie"/>
   </widget>
   <addaction name="menuPogoda"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUstawienia_programu">
   <property name="text">
    <string>Ustawienia programu</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionWyj_cie">
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
